bmwqemu.pm

```diff
@@ -40,10 +40,6 @@ our $screenshotpath = "qemuscreenshot";
 
 # global vars
 
-our $logger;
```

Comment:
I suppose you can remove the comment above as well.

Reply:
No, we still have global vars, e.g. %vars.